content/blog/tidymodels-2021-q2/index.Rmd

```diff
@@ -0,0 +1,232 @@
+
+Since our last roundup post, there have been 19 CRAN releases of 15 different packages. That might sound like a lot of change to absorb as a tidymodels user! However, we purposefully write code in small, modular packages that we can release frequently, to make models easier to deploy and our software easier to maintain. You can install these updates from CRAN with:
+
+```{r, eval = FALSE}
+install.packages(c("broom", "butcher", "discrim", "hardhat", "parsnip", "probably", 
+                   "recipes", "rsample", "stacks", "themis", "tidymodels", 
+                   "tidyposterior", "tune", "workflowsets", "yardstick"))
+```
+
+The `NEWS` files are linked here for each package; you'll notice that many of these releases involve small bug fixes or internal changes that are not user-facing. It's a lot to keep up with and there are some super useful updates in the mix, so read on for several highlights!
+
+- [broom](https://broom.tidymodels.org/news/index.html#broom-0-7-8-2021-06-24)
+- [butcher](https://butcher.tidymodels.org/news/index.html#butcher-0-1-4-2021-03-19)
+- [discrim](https://discrim.tidymodels.org/news/index.html#discrim-0-1-2-2021-05-28)
+- [hardhat](https://hardhat.tidymodels.org/news/index.html#hardhat-0-1-4-2020-07-02)
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-0-1-6-2021-05-27)
+- [probably](https://probably.tidymodels.org/news/index.html#probably-0-0-6-2020-06-05)
+- [recipes](https://recipes.tidymodels.org/news/index.html#recipes-0-1-16-2021-04-16)
+- [rsample](https://rsample.tidymodels.org/news/index.html#rsample-0-1-0-2021-05-08)
+- [stacks](https://github.com/tidymodels/stacks/blob/main/NEWS.md#v020)
+- [themis](https://themis.tidymodels.org/dev/news/index.html#themis-0-1-4-2021-06-12)
+- [tidymodels](https://tidymodels.tidymodels.org/news/index.html#tidymodels-0-1-3-2021-04-19)
+- [tidyposterior](https://github.com/tidymodels/tidyposterior/blob/master/NEWS.md#tidyposterior-010)
+- [tune](https://tune.tidymodels.org/news/index.html#tune-0-1-5-2021-04-23)
+- [workflowsets](https://workflowsets.tidymodels.org/news/index.html#workflowsets-0-0-2-2021-04-16)
+- [yardstick](https://yardstick.tidymodels.org/news/index.html#yardstick-0-0-8-2021-03-28)
+
+
+## Reduce the memory footprint of your recipes
+
+The [butcher](https://butcher.tidymodels.org/) package provides methods to remove (or "axe") components from model objects that are not needed for prediction. There are many methods for various kinds of models, but the most recent release updated how butcher handled _recipes_, the tidymodels' approach for preprocessing and feature engineering, for more complete and robust coverage. Let's consider a simulated churn classification data set for phone company customers:
```

Comment:
```suggestion
The [butcher](https://butcher.tidymodels.org/) package provides methods to remove (or "axe") components from model objects that are not needed for prediction. There are many methods for various kinds of models, but the most recent release updated how butcher handles _recipes_, the tidymodels' approach for preprocessing and feature engineering, for more complete and robust coverage. Let's consider a simulated churn classification data set for phone company customers:
```

Reply:
```suggestion
The [butcher](https://butcher.tidymodels.org/) package provides methods to remove (or "axe") components from model objects that are not needed for prediction. There are many methods for various kinds of models, but the most recent release updated how butcher handled _recipes_ (the tidymodels approach for preprocessing and feature engineering) for more complete and robust coverage. Let's consider a simulated churn-classification dataset for phone company customers:
```

Reply:
I think when you have the "the" in there for "the tidymodels approach" it's not possessive (it would be if it was "tidymodels' approach").

Also, in the following:
> There are many methods for various kinds of models

are you referring to literal R methods? I can't decide if it's one of those special words that we basically can't use in its more quotidian sense because of the context of "being about R." One option would be just to cut that first clause and begin with:
> The most recent release…

I don't think you'd lose much info.

"churn-classification" gets a hyphen just because it's modifying a noun (compound modifier)

Lastly, I think dataset has reached its evolution into closed-form (especially in the [technical context](https://english.stackexchange.com/a/2129/44117)). The two word (open form) -> compound word (hyphenated) -> one word (closed form) evolution is always up for debate (because, y'know, English be like that), but I go closed when it resolves ambiguity _especially_ when the latter term can also be parsed as a verb.